### PR TITLE
[CNXC-313] Update Footer

### PR DESCRIPTION
**Pull request by colinywu**

### CODE REVIEW

**Comment by boris-nguyen** [comment]
Overall, good work @colinywu 

1 small suggestion I have is consider adding flexbox to improve the responsiveness when the window is resized.

**Review by boris-nguyen — APPROVED** [review]

File: src/containers/Layout/Footer.tsx [hunk 1]
```diff
@@ -18,17 +18,18 @@ const Footer: React.FC<FooterProps> = ({ isLogin }) => {
         <PageHeaderToolsItem>
           <p className="footer-text">An open source clinical decision support UI platform created by&nbsp;
             <a href="https://www.darwinai.com/" target="_blank" rel="noopener noreferrer">DarwinAI</a>,&nbsp;
-            <a href="https://chrisproject.org/" target="_blank" rel="noopener noreferrer">ChRIS</a>, and&nbsp;
-            <a href="https://redhat.com/chris" target="_blank" rel="noopener noreferrer">RedHat</a>&nbsp;</p>
+            <a href="https://chrisproject.org/" target="_blank" rel="noopener noreferrer">BCH/ChRIS</a>, and&nbsp;
```
Comment (janakitti) [inline src/containers/Layout/Footer.tsx:21]:
```suggestion
            <a href="https://chrisproject.org/" target="_blank" rel="noopener noreferrer">BCH (The ChRIS Project)</a>, and&nbsp;
```

File: src/containers/Layout/Footer.tsx [hunk 2]
```diff
@@ -18,17 +18,18 @@ const Footer: React.FC<FooterProps> = ({ isLogin }) => {
...
         </PageHeaderToolsItem>
       </PageHeaderTools>
+      <span>|</span>
       <PageHeaderTools>
         <PageHeaderToolsItem className="footer-link">
           <p className="footer-text">
-            Find more about&nbsp;
+            Learn more about&nbsp;
             <a href="https://alexswong.github.io/COVID-Net/" target="_blank" rel="noopener noreferrer">COVID-Net</a>
             &nbsp;and&nbsp;
```
Comment (janakitti) [inline src/containers/Layout/Footer.tsx:31]:
```suggestion
            &nbsp;and the&nbsp;
```

File: src/containers/Layout/Footer.tsx [hunk 3]
```diff
@@ -18,17 +18,18 @@ const Footer: React.FC<FooterProps> = ({ isLogin }) => {
...
-            <a href="https://github.com/darwinai/covidnet_ui" target="_blank" rel="noopener noreferrer">COVID-Net UI</a>
+            <a href="https://github.com/darwinai/covidnet_ui" target="_blank" rel="noopener noreferrer">the COVID-Net UI</a>
```
Comment (janakitti) [inline src/containers/Layout/Footer.tsx:32]:
```suggestion
            <a href="https://github.com/darwinai/covidnet_ui" target="_blank" rel="noopener noreferrer">COVID-Net UI</a>
```

**Review by janakitti — CHANGES_REQUESTED** [review]

File: src/containers/Layout/Footer.tsx [hunk 4]
```diff
@@ -18,17 +18,18 @@ const Footer: React.FC<FooterProps> = ({ isLogin }) => {
         <PageHeaderToolsItem>
           <p className="footer-text">An open source clinical decision support UI platform created by&nbsp;
```
Comment (janakitti) [inline src/containers/Layout/Footer.tsx:19]:
```suggestion
          <p className="footer-text">An open source neural network platform created by&nbsp;
```
This was the suggestion from the call with FNNDSC

**Review by janakitti — APPROVED** [review]
Great work, @colinywu!

**Pull request merged**